src/modules/shared/database/migrations/0010_messy_vargas.sql

```
@@ -0,0 +1,6 @@
+CREATE INDEX "drafts_status_idx" ON "drafts" USING btree ("status");--> statement-breakpoint
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)